spring-jdbc/src/main/java/io/crate/spring/jdbc/samples/model/Image.java

```diff
@@ -0,0 +1,38 @@
+package io.crate.spring.jdbc.samples.model;
+
+import java.util.Date;
+import java.util.Objects;
+
+public class Image {
+
+    private final String digest;
+
+    private final Date last_modified;
```

Comment:
why do we have snake case field and method names in many places? If it related to serialization, i think there should be some spring/json mappers magic annotations that will map column names to object fields/